### PR TITLE
Update to dev branch joomla-testing-robo

### PR DESCRIPTION
**Pull request by 810**
Updating to the dev branch for the container-test branch. So we can use phpcs-2 with joomla-browser.

### CODE REVIEW

**Comment by yvesh** [comment]
@810 can't we make a new release? @jatitoam

**Comment by jatitoam** [comment]
We will but it's still ongoing since we still don't have all the features used by Weblinks RoboFile

**Comment by yvesh** [comment]
yep, no chance to make a release meanwhile? If possible we should not switch to dev branch :/

**Comment by jatitoam** [comment]
No worries, he's pushing to a feature (dev) branch too :)

**Comment by jatitoam** [comment]
However tests are still failing, I think this was just where I got stuck 2 weeks ago when I was working on it.

**Comment by 810** [comment]
because the old codestyle was loaded 
now I'm added the new one for phpcs-2. should be working now.

File: RoboFile.php [hunk 1]
```diff
@@ -19,15 +19,17 @@ public function checkCodestyle($sniffersPath = null)
 	{
 		if (is_null($sniffersPath))
 		{
-			$sniffersPath = __DIR__ . '/.travis/phpcs/Joomla';
+			$sniffersPath = __DIR__ . '/.travis/phpcs/';
 		}
 
 		$this->taskCodeChecks()
 			->setBaseRepositoryPath(__DIR__)
+			->setCodeStyleStandardsRepo('photodude/coding-standards')
```
Comment (jatitoam) [inline RoboFile.php:27]: You can use the provided `vendor/greencape/coding-standards/src` folder to fix this (no need to specify a branch)

File: composer.json [hunk 2]
```diff
@@ -19,11 +19,12 @@
...
+        "joomla-projects/joomla-testing-robo"   : "dev-container-test"
     },
     "autoload": {
         "psr-4": {
             "Codeception\\Module\\"             : "src"
         }
-    }
+    },
+    "minimum-stability": "dev"
 }
```
Comment (jatitoam) [inline composer.json:31]: I was missing this: please also add `"prefer-stable": true` since it's only using development versions without it

**Comment by 810** [comment]
Ok all codestyle has now been updated to the new rules of phpcs 2.

**Comment by jatitoam** [comment]
Great job @810!!

**Pull request merged**